GitHubReferences.ps1

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Please ignore this file, I'll remove the associated commit and rebase/squash as required after the GitHubReference changes are in